tests/functional/views/basic_lti_launch_test.py

```diff
@@ -16,30 +16,22 @@
 
 
 class TestBasicLTILaunch:
-    def test_requests_with_no_oauth_signature_are_forbidden(self, app, lti_params):
+    def test_requests_with_no_oauth_signature_are_forbidden(
+        self, lti_params, post_launch
+    ):
         del lti_params["oauth_signature"]
 
-        response = app.post(
-            "/lti_launches",
-            params=lti_params,
-            headers={
-                "Accept": "text/html",
-                "Content-Type": "application/x-www-form-urlencoded",
-            },
+        response = post_launch(
+            post_params=lti_params,
             status=403,
```

Comment:
If you delete the trailing comma from this line then Black will format the call onto one line for you. I sometimes wonder whether we should just disable Black's [magic trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) feature

Reply:
This is mostly me not re-visiting the commas after adding/remmoving params. 

I think we should keep black's default, it's very handy to control a bit the formatting when for example mixing with with `pylint: disable...` hints.